app/controllers/funding/base_funding_data_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Funding
+  class BaseFundingDataController < ApplicationController
```

Comment:
Maybe `Data` is not needed in this name?
```suggestion
  class BaseFundingController < ApplicationController
```